### PR TITLE
[test] test2

### PR DESCRIPTION
**Pull request by DawidPiechota**
# 

- [ ]  discussions are resolved
- [ ]  existing unit tests are passing
- [x]  new unit tests are created and passing
- [x]  code is autoformatted
- [x]  linting issues are addressed
- [x]  documentation from code is updated
- [x]  PR name follows the convention. Example: “[HDD-0001] Add “
- [ ] test

### CODE REVIEW

**Comment by bartosz-bok** [comment]
test from bartek

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # heart-disease-detector
-A program that calculates the probability of heart disease based on user responses to lifestyle questions 
+A program that calculates the probability of heart disease based on user responses to lifestyle questions.
```
Comment (bartosz-bok) [inline README.md:2]: tu jest z kropkĄĄ

**Review by bartosz-bok — CHANGES_REQUESTED** [review]
liv koment

File: README.md (hunk 1 above)
Comment (DawidPiechota) [reply README.md:2]: Bo ma być z kropką

Comment (bartosz-bok) [reply README.md:2]: aaaaaaaa, dobra

**Comment by bartosz-bok** [comment]
> Tutaj

- [ ] pierwszy test

**Pull request closed without merging**